tldr.py

```diff
@@ -18,10 +18,16 @@
 # Required for Windows
 import colorama
 
-DEFAULT_REMOTE = "https://raw.githubusercontent.com/tldr-pages/tldr/master/pages"
+DEFAULT_REMOTE = os.environ.get(
```

Comment:
I think it would be warranted to rename this to something like PAGES_SOURCE_LOCATION, for various reasons:

- Adding LOCATION to match the naming of DOWNLOAD_CACHE_LOCATION
- Removing REMOTE because the source does not need to be a remote one (unless you're interpreting the term "remote" from the point of view of this codebase; but I'd argue that most people would interpret "remote" to mean something that's not in the local machine, i.e. that needs to be fetched over the network)
- Removing DEFAULT because if this is configurable, then the value captured in the variable no longer is a default; what's the default is the URL that's used as the fallback.

WDYT?

Reply:
That said, it's not a deal breaker for this PR. I'd be OK with the PR being merged in its current state, so please consider this an "approval with comments".

Reply:
Well said @waldyrious, I agree with you.

Reply:
variable updated to `PAGES_SOURCE_LOCATION`, also renamed the environment variables to match the name of these variables.